.changeset/tiny-feet-enjoy.md

```diff
@@ -0,0 +1,6 @@
+---
+"@marigold/docs": patch
+"@marigold/components": patch
```

Comment:
Needs to be flagges as a breaking change!

Reply:
like thsi?